tuna/miopen/subcmd/export_db.py

```diff
@@ -49,8 +49,11 @@
 
 DIR_NAME = {'F': 'Fwd', 'B': 'BwdData', 'W': 'BwdWeights'}
 
-_, ID_SOLVER_MAP = get_id_solvers()
+ID_SOLVER_MAP = None
```

Comment:
Changes to this file are because it is included by tuna/lib_utils.py and the act of importing will call get_id_solvers() because it's at top level.  get_id_solvers() looks in the tuna.solvers table and barfs if there isn't one.  In the rocmlir case, there isn't one.  These changes arrange to not initialise ID_SOLVER_MAP until it is used, and I'm sure there's a more elegant way to do it.

Reply:
Still not ideal, and pylint didn't like it.  Adding a docstring and a global declaration (and a suppression) satisfied pylint.  I tried a singleton object but that was worse.